libs/adapters/src/knock/index.ts

```diff
@@ -110,11 +110,18 @@ export function KnockProvider(): NotificationsProvider {
     },
 
     async getSchedules(options): Promise<NotificationsProviderSchedulesReturn> {
-      const res = await knock.users.getSchedules(options.user_id, {
-        // @ts-expect-error Knock SDK doesn't support this option, but it works since the Knock API supports it
-        workflow: options.workflow_id,
-      });
-      return formatScheduleResponse(res.entries);
+      try {
+        const res = await knock.users.getSchedules(options.user_id, {
+          // @ts-expect-error Knock SDK doesn't support this option, but it works since the Knock API supports it
+          workflow: options.workflow_id,
+        });
+        return formatScheduleResponse(res.entries);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      } catch (e: any) {
```

Comment:
I was wondering if we could set

```
useUnknownInCatchVariables: false
```

in the libs/config.ts to override the strict mode default. This is the only rule I don't like becuase if we are creating our own Error type and not extending Error, then we are doing something wrong in my opinon.

Anyways @Rotorsoft @rbennettcw any thoughts on this?

Reply:
We still have a few places where we have the test guard  `err instanceof Error` to account for non Error errors. I'm ok with treating err as unknown, unless we are 100% sure that nothing is throwing non Error errors.

What we should do is some refactoring making sure we use utils that can encapsulate this error handling guard,  like for example change to log.error(err:unknown) instead of log.error(err:Error)